logger/logger.go

```diff
@@ -34,29 +40,45 @@ type LogInfo map[string]interface{}
 // Logger a logrus implementer of the JSON logger interface
 type Logger struct {
 	logger *logrus.Entry
+
+	supressionMessages []string
 }
 
 // NewLogrusLogger return a LogrusLogger
-func NewLogrusLogger(context string, info LogInfo, logger *logrus.Entry) *Logger {
+func NewLogrusLogger(context string, info LogInfo, logger *logrus.Entry) (*Logger, error) {
 	if logger == nil {
 		logger = logrus.WithField("context", context)
 	}
-
-	l := &Logger{logger: logger}
+	var suppressionMessages []string
+	l := &Logger{logger: logger, supressionMessages: make([]string, 0)}
 	l.logger = l.getEntry(info)
-	return l
+	value, ok := os.LookupEnv("SUPPRESSE_TLS_HANDSHAKE_ERROR_LOGGING")
+	if ok {
+		ok, err := strconv.ParseBool(value)
```

Comment:
How about `b` instead of `ok` here for the parsed boolean value? `ok` carries a slightly different meaning usually.